specs/opcode/54SLOAD_55SSTORE.md

```diff
@@ -0,0 +1,91 @@
+# SLOAD & SSTORE op code
+
+## Variables definition
+
+| Name | Value |
+| - | - |
+| COLD_SLOAD_COST | 2100 |
+| WARM_STORAGE_READ_COST | 100 |
+| SLOAD_GAS | 100 |
+| SSTORE_SET_GAS | 20000 |
+| SSTORE_RESET_GAS | 2900 |
+| SSTORE_CLEARS_SCHEDULE | 15000 |
+
+## Constraints
+
+1. opcodeId checks
+   1. opId === OpcodeId(0x54) for `SLOAD`
+   2. opId === OpcodeId(0x55) for `SSTORE`
+2. state transition:
+   - gc
+     - `SLOAD`: +5 (2 stack operations + 1 storage reads + 2 access_list reads/writes)
+     - `SSTORE`: +8
+       + 2 stack operations
+       + 2 storage reads/writes
+       + 2 access_list reads/writes
+       + 2 gas_refund reads/writes
```

Comment:
IIUC, `gas_refund` happen when reverting occur, right ?

Reply:
if reverted, then there will not be `gas_refund`. 

check `gas_refund` here: 
+ https://eips.ethereum.org/EIPS/eip-2200
+ https://eips.ethereum.org/EIPS/eip-2929
+ https://www.evm.codes/about

Reply:
I see, Thanks the reference !